_tuning-your-cluster/replication-plugin/getting-started.md

```diff
@@ -275,15 +275,15 @@ Note that you can't resume replication after it's been paused for more than 12 h
 When you no longer need to replicate an index, terminate replication from the follower cluster:
 
 ```bash
-curl -XPOST -k -H 'Content-Type: application/json' -u 'admin:admin' 'https://localhost:9200/_plugins/_replication/follower-01/_stop?pretty' -d '{}'
+curl -XPOST -k -H 'Content-Type: application/json' -u 'admin:< Admin password >' 'https://localhost:9200/_plugins/_replication/follower-01/_stop?pretty' -d '{}'
```

Comment:
```suggestion
curl -XPOST -k -H 'Content-Type: application/json' -u 'admin:<custom-admin-password>' 'https://localhost:9200/_plugins/_replication/follower-01/_stop?pretty' -d '{}'
```

Reply:
Applied suggestion in: [cdeb90f](https://github.com/opensearch-project/documentation-website/pull/5887/commits/cdeb90ffae8be7a3ae2f9de267abc08879dd44cd)